creator-node/src/routes/nodeSync.js

```diff
@@ -308,19 +308,23 @@ async function _nodesync (req, walletPublicKeys, creatorNodeEndpoint, dbOnlySync
           transaction
         })
         const fetchedLatestBlockNumber = fetchedCNodeUser.latestBlockNumber
+        const fetchedLatestClockVal = fetchedCNodeUser.clock
```

Comment:
get ack for this change from @SidSethi and @dmanjunath  
```
scenario where clocknumbers are different between primary and secondary users but blocknumbers are equal resulting in a scenario where snapback believes a sync is pending but the sync operation is rejected because of Error: Imported data is outdated, will not sync. Imported latestBlockNumber                 125 Self latestBlockNumber 125

primary DB record is
0x5a643b6c60eb40329484430f0d9bbf62388aef81	2020-10-08 03:34:47.586+00	2020-10-08 04:11:45.861+00	0bd97b06-1c58-4ad0-96d0-4d530a433088	125	186
(clock value 186, block processed 125)


secondaries have the following DB record:
0x5a643b6c60eb40329484430f0d9bbf62388aef81	2020-10-08 03:34:47.586+00	2020-10-08 04:05:09.009+00	0bd97b06-1c58-4ad0-96d0-4d530a433088	125	180
(clock value 180, block processed 125)
```

Reply:
ack'd, change completed post discussion